CHANGES.rst

```diff
@@ -4,7 +4,14 @@ Changelog for zest.releaser
 9.4.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Try hard to determine our own installed version, but do not fail when this does not work.
```

Comment:
Too much detail for a changelog entry? The three lines can be summarized as “Worked around python 3.9 version-reporting issue that prevented startup”?

Reply:
Agreed and done.